vscode-web/README.md

```diff
@@ -48,3 +48,20 @@ module "vscode-web" {
   accept_license = true
 }
 ```
+
+### Pre-configure Settings
+
+Configure VS Code's [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) file:
```

Comment:
Should we add a warning that user settings cannot be customized because they are stored in the browser, so only machine settings will work?  We could also recommend `code-server` if they absolutely need to have user settings, or recommend using settings sync.

Reply:
@code-asher - maybe for another PR, but i think it would be useful to list out know issues and benefits the differentiate to the options. 

ie: 
code-server:
-  is more functionally complete 
- has less issues
- does not support the microsoft marketplace
- does not support  some of the microsoft extensions like copilt, liveshare
- does not require wildcard urls
- etc..

vscode-web:
- has issues with port forwarding
- settings issues
- requires wildcard / subdomain = true.
- does not support self-hosted extensions
- supports the microsoft extensions
- etc..

Reply:
Agreed!  This is a good list.  code-server has a list of differences from Codespaces (https://github.com/coder/code-server/blob/main/docs/FAQ.md#whats-the-difference-between-code-server-and-github-codespaces) but not the self-hosted version, so we should probably add something there as well.  Some of the differences there apply to the self-hosted version as well though.